var/spack/repos/builtin/packages/pango/package.py

```diff
@@ -43,7 +43,6 @@ class Pango(AutotoolsPackage):
 
     depends_on("pkg-config", type="build")
     depends_on("harfbuzz")
-    depends_on("cairo")
```

Comment:
@adamjstewart I can put it back if you think it is necessary, but I just ran

spack spec pango^cairo@1.14.8

and it returned a concretized spec for me.